apps/emqx_auth_http/etc/emqx_auth_http.conf

```diff
@@ -151,3 +151,9 @@ auth.http.pool_size = 32
 ##
 ## Value: File
 ## auth.http.ssl.keyfile = {{ platform_etc_dir }}/certs/client-key.pem
+
+## In mode verify_none the default behavior is to allow all x509-path
+## validation errors.
+##
+## Value: verify_peer | verify_none
+## auth.http.ssl.verify = verify_peer
```

Comment:
## Turn on peer certificate verification
##
## Value: true | false
## web.hook.ssl.verify = false

Can the format be unified with webhook？